api/Updated-Metrics.md

```diff
@@ -0,0 +1,368 @@
+# Updates for Metrics
+
+This is an update for the existing metrics document, which is being left in place for now as a point of comparison.
+
+## Assumed data
+
+In the following we assume that we have variables of the following form defined:
+
+```python
+y_true = [0, 1, 0, 0, 1, 1, ...]
+y_pred = [1, 0, 0, 1, 0, 1, ...]
+A_sex = [ 'male', 'female', 'female', 'male', ...]
+A_race = [ 'black', 'white', 'hispanic', 'black', ...]
+A = pd.DataFrame(np.transpose([A_sex, A_race]), columns=['Sex', 'Race'])
+
+weights = [ 1, 2, 3, 2, 2, 1, ...]
+```
+
+We actually seek to be very agnostic as to the contents of the `y_true` and `y_pred` arrays.
```

Comment:
agnostic in what way? I think the proposal later explains this, but it may be easier to follow if that's moved up here.

Reply:
I was meaning in exactly the same way as your suggested comment. Let me rephrase.